pkg/operator/controllers/pullsecret/pullsecret_controller_test.go

```diff
@@ -775,7 +775,7 @@ func TestEnsureGlobalPullSecret(t *testing.T) {
 			utilerror.AssertErrorMessage(t, err, tt.wantError)
 
 			if !reflect.DeepEqual(s, tt.wantSecret) {
-				t.Fatalf(cmp.Diff(s, tt.wantSecret))
+				t.Fatalf("%s", cmp.Diff(s, tt.wantSecret))
```

Comment:
In all cases where the format functions had no format strings, simply use the non-formatted version. Here, for instance:

```go
t.Fatalf(cmp.Diff(s, tt.wantSecret))
```

Or, of course, actually add some context:

```go
t.Fatalf("invalid secret: %s", cmp.Diff(s, tt.wantSecret))
```

Reply:
Addressed